README.md

```diff
@@ -9,7 +9,7 @@ The development Qt (https://www.qt.io/) libraries are required to build the GUI
 ### Ubuntu
 
 ```bash
-$ sudo apt install qt5-default
+$ sudo apt install qt5-default libopenslide-dev
```

Comment:
The build failure is hard to find, I had to go from github to azure then download a compressed log file, but it eventually says:

> 2019-06-06T15:41:49.1869172Z -- Could NOT find Openslide (missing: OPENSLIDE_LIBRARIES OPENSLIDE_INCLUDE_DIRS) 

So libopenslide-dev needs to be added to the CI too.

Reply:
Thanks, I'll add that